internal/app/orchestrator/downstream.go

```diff
@@ -24,8 +25,8 @@ import (
 // WaitGroup is used here to ensure that all writes to the channel are
 // processed before we attempt to close the channel.
 type responseChannel struct {
-	wg      sync.WaitGroup
-	channel chan gcp.Response
+	wg sync.WaitGroup
+	w  transport.Watch
```

Comment:
can we explicitly call this `watch` or `transport`?

Reply:
its in a separate `transport` package. Are you suggesting keeping the watch in the orchestrator package?

Reply:
ah I just mean changing the variable name `w`.

Reply:
ah right...doing that now

Reply:
done